xcode/Subconscious/Shared/Library/NSRangeUtilities.swift

```diff
@@ -14,3 +15,26 @@ extension NSRange {
         return range != nil
     }
 }
+
+
+// https://gist.github.com/krzyzanowskim/057676af670f06fa6061473ac28a6c58
+extension NSRange {
```

Comment:
No license on this gist :( but it's incredibly useful

Reply:
If no license, then we must write our own.